### DISCUSSION TITLE
Transfer ElementaryTable object into dataframe, is that possible?

### DISCUSSION THREAD

**Discussion by dustreturn** [insightsengineering/rtables#967] [Q&A]
The reason I'm asking this question is to see if there is an easy way to output the results of rtable as a dataframe, to facilitate double programming QC (which is quite SAS programmer...).

**Comment by Melkiades** [comment]
Yes! It is possible. The general function is called `as_result_df`. Check the documentation, in particular, you might want to try the novel `make_ard = TRUE`, which produces a line per statistical output.
